modules/cpr/src/main/java/org/atmosphere/cpr/BroadcasterCache.java

```diff
@@ -84,6 +84,18 @@
      * and some future must be cancelled. This method will always be invoked when a {@link Broadcaster} gets destroyed.
      */
     void cleanup();
+    
+    /**
+     * Get serializer for cache messages.
+     * @return The {@link Serializer}.
+     */
+    Serializer getCacheSerializer();
```

Comment:
You don't need to add API on BroadcasterCache IMO. You can also retrieve the broadcaster.